apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx

```diff
@@ -126,7 +137,7 @@ export default class ModerationNotification extends Component {
         </div>
         <div className={'mt-3 d-flex justify-content-' + (!this.state.isPending && !this.state.isBlocked ? 'end' : 'between')}>
           {this.state.isPending
-            ? <><button className="btn btn--none" type="button"><i className="fas fa-reply" aria-hidden="true" />{replyText}</button>
+            ? <><button className="btn btn--none" type="button" onClick={this.toggleModerationNote.bind(this)}><i className="fas fa-reply" aria-hidden="true" />{replyText}</button>
```

Comment:
any reason not to note this handler same as `this.toggleIsBlocked()` and `this.toggleIsPending()`?

something like `(e) => this.toggleModerationNote(e)` should work the same way.
I kind of avoid the `.bind(this)`, because it always looked to me like a workaround (i know it is legacy javascript style, but i think the arrow notation is more readable and we use it below).

Reply:
first question: not that I know of.
second question: yeah I am having trouble adapting to arrow notation as I find it less readable, will work on it when we come back to it though, cheers!

Reply:
@khamui see new PR https://github.com/liqd/a4-kosmo/pull/173